shared/src/formationData.ts

```diff
@@ -1,16 +1,56 @@
+import { CountriesShortCodes } from "./countries";
 import { Municipality } from "./municipality";
+import { StateNames, StateObject } from "./states";
 
-export const defaultFormationLegalType: FormationLegalType = "limited-liability-company";
+export const defaultFormationLegalType: PublicFilingLegalType = "limited-liability-company";
 
-export const FormationLegalTypes = [
+export type SignerTitle =
+  | "Authorized Representative"
+  | "Authorized Partner"
+  | "Incorporator"
+  | "General Partner"
+  | "President"
+  | "Vice-President"
+  | "Chairman of the Board"
+  | "CEO";
+
+export const publicFilingLegalTypes = [
   "limited-liability-partnership",
   "limited-liability-company",
   "limited-partnership",
   "c-corporation",
   "s-corporation",
 ] as const;
 
-export type FormationLegalType = typeof FormationLegalTypes[number];
+export type PublicFilingLegalType = typeof publicFilingLegalTypes[number];
+
+export const formationLegalTypes = [
```

Comment:
just a style thing, but when I have a list like this, I'd rather name it something like "allFormationLegalTypes" to make it clear it's a list of all the elements.

Otherwise you're expecting developers to know the difference between `formationLegalTypes` and `FormationLegalType` which is a very subtle naming distinction for two things that are used very differently